radicle-cli/src/commands/assign.rs

```diff
@@ -1,31 +1,32 @@
 use std::ffi::OsString;
-use std::str::FromStr;
 
 use anyhow::anyhow;
 use nonempty::NonEmpty;
-use radicle::prelude::Did;
 
-use crate::terminal as term;
-use crate::terminal::args::{Args, Error, Help};
 use radicle::cob;
 use radicle::cob::issue;
+use radicle::prelude::Did;
 use radicle::storage::WriteStorage;
 
+use crate::terminal as term;
+use crate::terminal::args::{Args, Error, Help};
+
 pub const HELP: Help = Help {
     name: "assign",
     description: "Assign an issue",
     version: env!("CARGO_PKG_VERSION"),
     usage: r#"
 Usage
 
-    rad assign <issue> --to <did>
+    rad assign <issue> --to <did> [<option>...]
 
     To assign multiple users to an issue, you may repeat
     the `--to` option.
 
+    --to <did>    Assignee to add to the issue
+
 Options
 
-    --to <did>    Assignee to add to the issue
```

Comment:
?

Reply:
This is moved above `Options` as this is not really optional, but something critical to the command. Looking over other `HELP` messages, `Options` mostly seem to be generic flags for a command, not those directly needed/affecting its logic.

You're confusion is justified as there's no other case where we directly describe the (critical) flag, but there are cases where there are hints on how to use them in the description text above `Options`.

Reply:
and if we want to keep it there, I suggest we change the command format from:
```
rad assign <issue> --to <did>
```

to

```
rad assign <issue> [<option>...]
```

but since `--to` is mandatory for the command, this is confusing.